src/theme/components.js

```diff
@@ -11,6 +11,12 @@ const components = {
           lineHeight: '1.4375em',
           padding: '16.5px 14px',
         },
+        '&.compare-button': {
+          lineHeight: '1.4375em',
+          padding: '16.5px 14px',
+          textTransform: 'uppercase',
```

Comment:
@julienw added this as adviced, (similar question to what I direct messaged you).
Is this the way you were suggestng? If not did you have a better way in mind ?